### PR TITLE
[stash.rb] adding sheaths to stash / empty_hands

### PR DESCRIPTION
**Pull request by OSXLich-Doug**

### CODE REVIEW

**Comment by LeBigNasty** [comment]
should probably be versioned

File: lib/stash.rb [hunk 1]
```diff
@@ -52,6 +54,23 @@ def self.add_to_bag(bag, item)
       end
     end
 
+    def self.sheath_bags
+      # find ready list settings for sheaths only; regex courtesy Eloot
+      @sheath = {}
+      @checked_sheaths = false
+      sheath_list_match = /(?:sheath|secondary sheath):\s+<d\scmd="store\s(\w+)\sclear">[^<]+<a\sexist="(\d+)"\snoun="[^"]+">([^<]+)<\/a>(?:\s[^<]+)?<\/d>/
+
+      ready_lines = Lich::Util.quiet_command_xml("ready list", /Your current settings are/)
```
[the remark this answers is not in the document]
Comment (mrhoribu) [reply lib/stash.rb:63]: Full output of the `READY LIST` output for debug/info comparison:
```xml
<!-- CLIENT --><c>ready list<!-- ENDCLIENT -->
<output class="mono"/>
Your current settings are:

  shield: (<d cmd='ready SHIELD'>none</d>) (<d cmd='store set'>put in sheath</d>)
  weapon: a lacquered illthorn runestaff (<d cmd='ready WEAPON clear'>not found</d>) (<d cmd='store set'>put in sheath</d>)
  secondary weapon: (<d cmd='ready 2WEAPON'>none</d>) (<d cmd='store set'>put in sheath</d>)
  ranged weapon: (<d cmd='ready RANGED'>none</d>) (<d cmd='store set'>stowed</d>)
  ammo bundle: (<d cmd='ready AMMO'>none</d>) (<d cmd='store set'>stowed</d>)
  ammo2 bundle: (<d cmd='ready AMMO2'>none</d>)
  sheath: <d cmd="store SHEATH clear">a <a exist="103506213" noun="harness">tooled dark leather harness</a> embroidered with veniom threaded runes</d>
  secondary sheath: (<d cmd='store 2SHEATH'>none</d>)

To change your default item for a category that is already set, clear the category first by clicking on the item in the list above.  Click <d cmd="ready list">here</d> to update the list.
<output class=""/>
<prompt time="1687793039">&gt;</prompt>
```

[the remark this answers is not in the document]
Comment (OSXLich-Doug) [reply lib/stash.rb]: Submitted change to align with recommended `end_prompt`.  Also moved to `issue_command`.

**Review by mrhoribu — APPROVED** [review]
So far working fine for me.

**Pull request merged**